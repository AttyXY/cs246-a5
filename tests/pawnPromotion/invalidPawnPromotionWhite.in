setup
+ K a1
+ k h8
+ P a7
done
game human human
move a7 a8 P    
move a7 a8 X
move a7 a8 K
move a7 a8 B
quit
quit
