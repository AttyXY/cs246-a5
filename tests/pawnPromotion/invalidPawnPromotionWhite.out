WELCOME TO CHESS! PLEASE ENTER ONE OF THE FOLLOWING COMMANDS TO BEGIN:
- setup
- game [player1] [player2]

STARTING SETUP

8  --------
7  --------
6  --------
5  --------
4  --------
3  --------
2  --------
1  K-------
   abcdefgh

8  -------k
7  --------
6  --------
5  --------
4  --------
3  --------
2  --------
1  K-------
   abcdefgh

8  -------k
7  P-------
6  --------
5  --------
4  --------
3  --------
2  --------
1  K-------
   abcdefgh

STARTING GAME!

8  -------k
7  P-------
6  --------
5  --------
4  --------
3  --------
2  --------
1  K-------
   abcdefgh

Invalid pawn promotion type.
Invalid pawn promotion type.
Invalid pawn promotion type.
8  B------k
7  --------
6  --------
5  --------
4  --------
3  --------
2  --------
1  K-------
   abcdefgh

Final Score:
White: 0
Black: 0
THANKS FOR PLAYING CHESS!
